party: Add a hook before the application of a party replacement

xungongcoopengo opened the pull request:
Fix #PROCK-1699

PR coog: https://github.com/coopengo/coog/pull/10229

JCavallo commented on tryton/gui/window/view_form/view/list_gtk/widget.py:
Je verrais bien un commentaire pour expliquer pourquoi tout ça :)

JCavallo commented:
Ajouter la référence vers la fiche bugs.tryton.org dans le message de commit

nicoe committed:
[6.8-patches] Don't reset dates to None when the date is invalid (#13)

PCLAS-209

nicoe committed:
[6.8-patches] Set translation source from the po file when importing the translation [PREVIEW] (#17)

JCavallo committed:
[trytond] Restore previous behaviour for locking [CUSTOM]

JCavallo committed:
Proper fix for commission dates [CUSTOM]

gregory-lecomte committed:
CI: Update gitlab pipeline [PREVIEW] (#39)

nicoe committed:
Remove extra methods disconnect_signals [CUSTOM] (#40)

JCavallo committed:
Add missing __version_coog__ for client builds [CUSTOM]

JCavallo committed:
Update translations

nicoe committed:
Use custom date parsing function [CUSTOM] (#42)

Fix PCLAS-282

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
trytond: Override default configuration [CUSTOM] (#41)

Fix PCLAS-294

Commit:
Packaging coog-23.38.2338

CoopVincent committed:
[CUSTOM] sourceeditor: Send a correctly formatted object to pyflakes' Message constructor

Fix #PJAZZ-1258

JCavallo committed:
Fix token authentication for 6.8 [CUSTOM]

JCavallo committed:
Add missing fr translation [CUSTOM]

JCavallo committed:
tryton/data/locale/fr/LC_MESSAGES/tryton.mo

Add compiled fr translation catalog [CUSTOM]

Commit:
Packaging coog-23.39.2339

JCavallo committed:
sao: Fix missing var initialization [CUSTOM]

nicoe committed:
Block row selected handler when unselect all of list-form [PREVIEW] (#46)

JCavallo committed:
merge 6.8: Fix pool preloading [PREVIEW] (#43)

Real fix will probably be https://foss.heptapod.net/tryton/tryton/-/issues/11401#note_331051

Fix PCLAS-300

Commit:
Packaging coog-23.40.2340

nicoe committed:
Add INDEX creation SQL dump on --no-indexes switch [CUSTOM] (#49)

Fix #PCLAS-309

JCavallo committed:
Log slow postgres queries [CUSTOM]

CoopVincent committed:
ryton: Fix missing search and replace translations [CUSTOM] (#47)

Fix #PJAZZ-1268

nicoe committed:
Add missing record arguments from get_eval [PREVIEW] (#57)

Commit:
party: Fix separator check [CUSTOM] (#58)

Fix #PMETA-422

JCavallo committed:
client: Add debugging tooltips [CUSTOM] (#61)

Fix PCLAS-355

JCavallo committed:
Add rec_name_key to ignored cache context keys [CUSTOM] (#59)

Fix PCLAS-350

nicoe committed:
Fix invalid entry CSS (#60)

nicoe committed:
Support database cache as template for tests on PostgreSQL [PREVIEW] (#55)

Commit:
Packaging coog-23.43.2343

nicoe committed:
Use postgresql://name to prefix test template names [CUSTOM] (#62)

PCLAS-359

nicoe committed:
Share record's _ids when saving new records [PREVIEW] (#65)

PCLAS-349
https://foss.heptapod.net/tryton/tryton/-/merge_requests/881

nicoe committed:
Use multiple jobs to dump and restore a test database [PREVIEW] (#64)

PCLAS-333
https://foss.heptapod.net/tryton/tryton/-/merge_requests/747

Commit:
Packaging coog-23.44.2344

nicoe committed:
Trigger search on search entry focus-out [CUSTOM] (#66)

PCLAS-363

nicoe committed:
Use fallback value for charset, url_charset and encoding_errors [PREVIEW] (#56)

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
tryton: Honor sequence when on_change adds new keys in a dict [CUSTOM] (#72)

Fix PCLAS-375

Commit:
Packaging coog-23.45.2345

JCavallo committed:
trytond: Reparse authorization header in token / bearer mode [CUSTOM] (#79)

Fix PCLAS-401

nicoe committed:
Set record as modified when a user clicks replace / replace all [CUSTOM] (#67)

PCLAS-320

nicoe committed:
Localize tab_domain expression when generating URL [CUSTOM] (#70)

PCLAS-373

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
tryton: Fix display of dict fields following 196fafd0cc (#80)

Fix PCLAS-404

nicoe committed:
Add Log Out action (#71)

PCLAS-357

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

nicoe committed:
Add setup function to activate_modules [CUSTOM] (#78)

PCLAS-392

CoopVincent committed:
fix titlebar render when minimized (#81)

Fix #PJAZZ-1440

JCavallo committed:
trytond: Disable cache threads during uWSGI init (#85)

Fix PCLAS-407

nicoe committed:
Report connection errors on the first contact [CUSTOM] (#84)

PCLAS-405

Commit:
Packaging coog-23.46.2346

JCavallo committed:
proteus: Disable cache thread while initializing [CUSTOM] (#86)

Fix PCLAS-410

JCavallo committed:
tryton: Do not use |=, we still build with 3.8 [CUSTOM]

JCavallo committed:
trytond: Do not COMMENT in a readonly Transaction [CUSTOM] (#87)

Fix PCLAS-412

xungongcoopengo committed:
tryton: Change color for tryton-bookmarks and tryton-menu svg [CUSTOM]

Fix #PJAZZ-1458

Commit:
Packaging coog-23.47.2347

JCavallo committed:
sao: Fix widget / icons / buttons alignment in theme [CUSTOM] (#91)

Fix PCLAS-425

JCavallo committed:
trytond: Use environment variables for saml configuration (#88)

Fix PCLAS-420

Commit:
Packaging coog-23.48.2347

JCavallo committed:
trytond: Add "notify" callbacks registered in the pool [CUSTOM] (#92)

Fix PCLAS-429

JCavallo committed:
sao: Iteration on Fix widget / icons / buttons alignment in theme [CUSTOM] (#91)

nicoe committed:
Use a custom error when the database is not available [PREVIEW] (#98)

PCLAS-386
https://foss.heptapod.net/tryton/tryton/-/issues/12736

nicoe committed:
Keep dictionary entries sorted when updating the value [PREVIEW] (#83)

PCLAS-389
https://foss.heptapod.net/tryton/tryton/-/issues/12750

JCavallo committed:
trytond: Log unauthorized cors [CUSTOM]

Commit:
Packaging coog-23.49.2349

nicoe committed:
Use only one commit during the upgrade process [PREVIEW] (#69)

PCLAS-370
https://foss.heptapod.net/tryton/tryton/-/issues/12596

nicoe committed:
Allow to activate stat thread by configuration [CUSTOM] (#100)

PCLAS-435

nicoe committed:
Remove demo database fallbacks [CUSTOM] (#101)

PCLAS-438

Commit:
Packaging coog-23.50.2350

nicoe committed:
Add timeit module to python's embedded packages in the exe (#103)

PCLAS-495

nicoe committed:
Mark M2M records for addition in parent after it has been saved [PREVIEW] (#106)

PCLAS-492
https://foss.heptapod.net/tryton/tryton/-/issues/12832

BeatriceSchuster committed:
trytond: Add keys in the server context to identify when a module is being installed inside a test [CUSTOM] (#93)

Fix #PJAZZ-1416

CoopVincent committed:
trytond: Fix translation of the translated fields message #107 

Fix #PJAZZ-1572

Commit:
Packaging coog-23.51.2351

Commit:
Packaging coog-23.52.2352

Commit:
Packaging coog-24.01.2401

justine-senellart committed:
tryton: Fix bug when a quick_action does not have an icon (#109)


#Fix #PMETA-755

Commit:
Packaging coog-24.02.2402

nicoe committed:
Allow to define a sao mount point [PREVIEW] (#112)

PCLAS-477

nicoe committed:
Check for the existence of a screen before restoring wizard's dialog window [PREVIEW] (#118)

PCLAS-518
https://foss.heptapod.net/tryton/tryton/-/issues/12944

nicoe committed:
Add a 'valid' property on Date / Time widget [PREVIEW] (#99)


Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
cron: Log cron overlap as warnings [CUSTOM] (#117)

Fix PCLAS-519

Commit:
Packaging coog-24.03.2403

JCavallo committed:
trytond: Always use new types in the pool [PREVIEW] (#119)

Fix PCLAS-542

JCavallo committed:
trytond: Iteration on 9cf05986d26878df9ec487f70a1be7e9b21ee588 (#123)

Fix PCLAS-552

nicoe committed:
Prevent StringPartitioned to evaluate its argument on instanciation [CUSTOM] (#120)

PCLAS-540

nicoe committed:
Use "display: grid" for widget positioning [PREVIEW] (#53)

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

BeatriceSchuster committed:
tryton: Add missing translation for "Attachments" [CUSTOM] (#114)

Fix #PJAZZ-1640

nicoe committed:
Add __hash__ implementation on StringPartitioned [CUSTOM] (#124)

nicoe committed:
Do not compute wizard size twice [PREVIEW] (#111)

PCLAS-511

JCavallo committed:
sao: Add custom.js in built images [CUSTOM] (#122)

Fix PCLAS-550

nicoe committed:
Replace warning popup by a log message (#121)

* Add menu entries and trick to open configuration directory [CUSTOM]

* Replace warning popup by a logging message [CUSTOM]

PCLAS-531

Commit:
Packaging coog-24.04.2404

JCavallo committed:
sao: Add missing image

nicoe committed:
Apply overflow value to every parent group of node [PREVIEW] (#126)

PCLAS-524

nicoe committed:
Filter history using SQL [PREVIEW] (#104)

PCLAS-331

JCavallo committed:
trytond: Restore support of _datetime_exclude (#128)

Fix PCLAS-583

JCavallo committed:
tryton: Uniquify log file names, rotate on 10 [CUSTOM] (#127)

Fix PCLAS-579

xungongcoopengo committed:
debug: remove history table [CUSTOM] (#129)

Commit:
Packaging coog-24.05.2405

JCavallo committed:
trytond: Allow to ilike on a None value [PREVIEW] (#132)

Fix PCLAS-608

Commit:
Packaging coog-24.06.2406

Commit:
Packaging coog-24.07.2407-rc1

nicoe committed:
Do not use python 3.10 idioms [CUSTOM] (#136)

PCLAS-622

Commit:
Packaging coog-24.08.2408-rc1

mamoun-rais committed:
trytond: Change french translation for menu Fields Access and field Create Access [CUSTOM] (#137)

* Fix #CCOOG-474

gregory-lecomte committed:
Update win-make.sh after first installation to build coog after successful tests (#134)

JCavallo committed:
sao: Full custom theme overhaul [CUSTOM] (#140)

Fix PCLAS-629

nicoe committed:
Add right click handling on trees [CUSTOM] (#141)

Fix PCLAS-618

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

nicoe committed:
Use a pure CSS solution to compute the container view height and width [PREVIEW] (#130)

Fix PCLAS-568

Commit:
Packaging coog-24.09.2409-rc1

BeatriceSchuster committed:
trytond: Fix translations [CUSTOM] (#142)

Fix #PJAZZ-1795
Fix #PJAZZ-1801

nicoe committed:
Do not set current record on Screen.load [CUSTOM] (#144)

PCLAS-612

nicoe committed:
Hide remove button of dict field when using no_command [CUSTOM] (#145)

PCLAS-558

nicoe committed:
Allow to override empty string representation in Selection fields [CUSTOM] (#147)

PCLAS-427

JCavallo committed:
sao: Use 'menu-like' button for main tab title (#143)

Fix PCLAS-636

JCavallo committed:
trytond: Force explicit usage of keywords in TableHandler [CUSTOM] (#148)

Fix PCLAS-645

JCavallo committed:
sao: Add custom.css to build [CUSTOM] (#146)

Fix PCLAS-639

JCavallo committed:
sao: Context menu open relate from all selected records [CUSTOM] (#151)

Fix PCLAS-663

JCavallo committed:
sao: Better scrollbars [CUSTOM] (#150)

Fix PCLAS-656

Commit:
Packaging coog-24.10.2410-rc1

JCavallo committed:
trytond: Add a way to force error in the error handler [CUSTOM] (#149)

Fix PCLAS-653

JCavallo committed:
sao / tryton: Naked click opens relations in new tabs (#152)

Fix PCLAS-617

PierreCookie committed:
backend: Remove Identifier from index files output (#156)

PROCK-????

nicoe committed:
Execute user login's actions sequentially [PREVIEW] (#159)

Fix #PCLAS-614
https://bugs.tryton.org/issue13074

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
sao: Remove shadowing of disabled buttons [CUSTOM] (#155)

Fix PCLAS-668

JCavallo committed:
sao: Handle quick-actions [CUSTOM] (#154)

Fix PCLAS-664

JCavallo committed:
sao: nil-check tabs before setting style [PREVIEW]

Commit:
Packaging coog-24.11.2411-rc1

JCavallo committed:
sao: Update icon and name [CUSTOM] (#162)

Fix PCLAS-681

JCavallo committed:
trytond: Add final migration hooks [CUSTOM] (#163)

Fix PCLAS-686

JCavallo committed:
trytond: Give access to options in final migrations [CUSTOM]

Fix PCLAS-686

JCavallo committed:
sao: Remove info messages after 3s [CUSTOM] (#168)

Fix PCLAS-701

nicoe committed:
Instantiate PYSON.Bool expression with new from PYSON.Not instances [PREVIEW] (#160)

Fixes #PCLAS-674
https://foss.heptapod.net/tryton/tryton/-/issues/13096

jmousset committed:
backend: allow testing Coog with sqlite backend (#68)

Commit:
Packaging coog-24.12.2412-rc3

JCavallo committed:
sao: Iteration on theme [CUSTOM] (#167)

Fix PCLAS-692

JCavallo committed:
sao: Use a spinner to show the user that something is running [CUSTOM] (#166)

Fix PCLAS-690

nicoe committed:
Use display flex to prevent overflow of legend in the group's fieldset [CUSTOM] (#171)

Fix PCLAS-709

nicoe committed:
Hide notebooks and groups when they are empty [PREVIEW] (#153)

Fixes PCLAS-660
https://bugs.tryton.org/issue13070

nicoe committed:
Use a grid to display Dict fields [PREVIEW] (#172)

Fix PCLAS-695

nicoe committed:
Use dotted read in the clients [PREVIEW] (#108)

Fix PCLAS-279
https://bugs.tryton.org/12856

JCavallo committed:
sao: Increase size of "small" wizards [CUSTOM] (#174)

Fix PCLAS-703

JCavallo committed:
sao / gtk: Do not trigger an extra read for an empty rec_name [CUSTOM] (#170)

Fix PCLAS-711

JCavallo committed:
sao: Re-enable quick-action button after completion (#169)

Fix PCLAS-707

JCavallo committed:
Revert "Use dotted read in the clients [PREVIEW] (#108)"

This reverts commit d649d486f4d54f2cbe62d2bad884d04f7cc9698d.

JCavallo committed:
sao: Escape calculations in less

JCavallo committed:
sao: Set default search limit to 100 [CUSTOM]

JCavallo committed:
Reapply "Use dotted read in the clients [PREVIEW] (#108)"

This reverts commit 61421f3715996ad6657bbdd8676057c321faa9d9.

nicoe committed:
Only trigger modified notification explicitely in O2M fields [CUSTOM] (#176)

Fix PCLAS-279

JCavallo committed:
trytond: Deactivate related_limit [PREVIEW]

JCavallo committed:
gtk: Do not use a global variable as default for dict fields (#177)

Fix PCLAS-724

JCavallo committed:
gtk: Accept dsm prototype in Timedelta constructor [PREVIEW] (#178)

Fix PMETA-1445

JCavallo committed:
gtk: Force py38 compatibility [CUSTOM] (#179)

JCavallo committed:
sao: Dotted read iteration [PREVIEW]

JCavallo committed:
sao: Do not try to load xxxx:string fields [CUSTOM]

JCavallo committed:
sao / gtk: iteration on dotted read [PREVIEW]

JCavallo committed:
sao: Fix iteration on rpc_entries [PREVIEW]

JCavallo committed:
sao: Add slight color on selected list-form [CUSTOM]

Commit:
Packaging coog-24.13.2413-rc3

JCavallo committed:
sao: Disable yexpand for now [CUSTOM]

JCavallo committed:
sao: Make selections without any values readonly [CUSTOM] (#180)

Fix PCLAS-734

nicoe committed:
Limit the search of NULL values in TableHandler's not_null_action (#175)

nicoe committed:
Return a promise when executing an action [PREVIEW] (#182)

Fix PCLAS-614

JCavallo committed:
sao: Properly update group.load api calls [CUSTOM]

nicoe committed:
Select first record in tree views [CUSTOM] (#181)

Fix PCLAS-726

Commit:
Packaging coog-24.14.2414-rc1

JCavallo committed:
sao: Do not expand invisible widgets (#184)

Fix PCLAS-753

nicoe committed:
Add ellipsis on column header [CUSTOM] (#185)

Fix PCLAS-597

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

nicoe committed:
Update column visibility at the end of rendering the treeview [PREVIEW] (#187)

For performance reasons it's better not to mix DOM reading and
modifying especially in JS loops.

PCLAS-772

AdrienBenduc committed:
Update translations

Commit:
Packaging coog-24.15.2415-rc1

nicoe committed:
Select first record of treeview if none is selected [CUSTOM] (#189)

Select first record from treeview if none is selected

PCLAS-767

nicoe committed:
Use a TableHandler method to fetch the estimated count of records [PREVIEW] (#191)

Use a TableHandler method to fetch the estimated count of records [PREVIEW]

https://bugs.tryton.org/13118
PCLAS-784

nicoe committed:
Execute user login's actions sequentially [PREVIEW] (#190)

https://bugs.tryton.org/13074
Fix PCLAS-614

JCavallo committed:
Iteration on fbf91cdc0f: handle table queries

nicoe committed:
Correctly use COUNT [CUSTOM] (#193)

Commit:
Packaging coog-24.16.2416-rc2

nicoe committed:
Force scrollbar inside xxx2Many fields [PREVIEW] (#199)

Force scrollbar inside xxx2Many fields [PREVIEW]

Fix PCLAS-693

nicoe committed:
Transfer configuration, profiles and plugins from previous version [PREVIEW] (#50)

Transfer configuration, profiles and plugins from previous version [PREVIEW]

---------

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

nicoe committed:
Add close button to "Filters" popup [CUSTOM] (#205)

Fix #CCOOG-642

nicoe committed:
Do not set overflow hidden on first column [PREVIEW] (#204)

Fix PCLAS-824

nicoe committed:
Restore attachment preview functionality [PREVIEW] (#206)

Fix PCLAS-820

nicoe committed:
Use the configured language when parsing XML files [PREVIEW] (#201)

https://bugs.tryton.org/13204
Fix PCLAS-816

BeatriceSchuster committed:
tryton: Change the translation of the 'show plain text' checkbox message [CUSTOM] (#188)

Fix #PJAZZ-2008

Commit:
Packaging coog-24.17.2417

Commit:
Packaging coog-24.18.2418-rc2

nicoe committed:
Fix size of char inputs when the size is set [PREVIEW] (#202)

FIX PCLAS-742

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

Commit:
Packaging coog-24.20.2419

JCavallo committed:
sao: Fix layout of tab wizards (#208)

Fix PCLAS-843

Commit:
Packaging coog-24.21.2421-rc1

nicoe committed:
Fetch name of wizards without a model [PREVIEW] (#207)

Fix PCLAS-770

nicoe committed:
Synchronize index creation method with upstream [PREVIEW] (#211)

Fix PCLAS-841

nicoe committed:
Optimize instance creation in __getattr__ [PREVIEW] (#210)

* Optimize instance creation in __getattr__ [PREVIEW]

Accessing a M2O field from a X2M field could result in as many
instantiations as there are records in the X2M field. It could be a
performance bottleneck due to the many creations of Model kwargs and
potential useless calls of transaction.set_context().

Thus the optimization is twofold:

 - Create an additional code path when the M2O field modifies the
   context
 - Cache the kwargs creation process to prevent extra calls resulting in
   the same dictionary

* Use read to fetch reverse field value of O2M field in get [PREVIEW]

Using a browse there in order to get the id is a performance bottleneck
as it will result in the instanciation of potentially thousands of
models.

---------

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

nicoe committed:
Allow to resize attachment preview panel [preview (#209)

https://bugs.tryton.org/13234
Fixes PCLAS-821

JCavallo committed:
trytond: Pass ids to read [PREVIEW] (#213)

Ref PCLAS-838

nicoe committed:
Use JSONB to store MultiSelection field on PostgreSQL [PREVIEW] (#203)

* Use JSONB to store MultiSelection field on PostgreSQL [PREVIEW]

https://bugs.tryton.org/11513
Closes PCLAS-818

* Allow migration

---------

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
trytond: Revert getattr optimization

Unfix PCLAS-838

nicoe committed:
Skip translations refering to unknown XML ID or unactivated source (#214)

Fix PCLAS-572

nicoe committed:
Correctly compute column visibility for buttons (#212)

Fix PCLAS-828

JCavallo committed:
Revert "Skip translations refering to unknown XML ID or unactivated source (#214)"

This reverts commit 8c0b2d71f5e202dfd22ac94b494915ddbb0f0085.

xungongcoopengo committed:
prock-1699

xungongcoopengo commented on modules/party/party.py:
Je ne veux pas beaucoup toucher tryton pour ce comportement, parce que l'endorsement est pure en coog

xungongcoopengo commented on modules/party/party.py:
Je voulais mettre NotImplementError, mais ce modèle est surchargé dans coog pas forcément avec endorsement installé, donc je mets return ici, est ce que c'est ok pour toi? @JCavallo

nicoe committed:
Optimize instance creation in ModelStorage.__getattr__  (#219)

* Optimize instance creation in ModelStorage.__getattr__ [PREVIEW]

https://bugs.tryton.org/13245

* Sort X2M fields according to the order [PREVIEW]

Fix PCLAS-889
https://bugs.tryton.org/13288

JCavallo committed:
trytond: Correctly override _convert_domain [CUSTOM] (#221)

Fix PCLAS-901

nicoe committed:
Alert when using underspecified relation in domain [PREVIEW] (#217)

Fix PCLAS-874

---------

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

Commit:
Packaging coog-24.22.2422

Commit:
Packaging coog-24.23.2423-rc2

nicoe committed:
Correctly compute column visibility [CUSTOM] (#226)

nicoe committed:
Add a No-op context manager [CUSTOM] (#225)

Fix PCLAS-854

JCavallo committed:
debug: Improve views, fix generation

Fix PCLAS-946

nicoe committed:
Do not reset SQL indexes on each __setup__ call [PREVIEW] (#227)

https://bugs.tryton.org/13299
Fix PCLAS-880

Commit:
Packaging coog-24.24.2424-rc3

xungongcoopengo committed:
Merge remote-tracking branch 'origin/master' into prock-1699

nicoe committed:
Order xxx2Many targets by id origin instead of order name [PREVIEW] (#231)

https://bugs.tryton.org/12795
Fix PCLAS-950

JCavallo committed:
sao: Hide nested checkbox for selecting lines in tree (#232)

Fix PCLAS-964

nicoe committed:
Consider that records in exception are loaded [PREVIEW] (#229)

* Always raise RPCException in RPCExecutre [PREVIEW]

* Consider that records in exception are loaded [PREVIEW]

https://bugs.tryton.org/13331

Fix PCLAS-945

* Raise an exception when it occurs when loading a record [PREVIEW]

https://bugs.tryton.org/13332

xungongcoopengo commented on modules/party/party.py:
selon Jean, hook devrait être after le remplacement

nicoe committed:
Do not uglify sao's code [CUSTOM] (#233)

JCavallo committed:
sao: Build using npm exec [CUSTOM]

PierreCookie committed:
sao: Force submodule dependency version (#234)

Fix #00000

JCavallo committed:
Skip translations refering to unknown XML ID or unactivated source [PREVIEW] (#215)

Fix PCLAS-883

---------

Co-authored-by: Nicolas Évrard <nicoe@openhex.org>

xungongcoopengo committed:
test

Commit:
Packaging coog-24.25.2425-rc2

JCavallo committed:
trytond: Disable not null constraint drop for sqlite [CUSTOM] (#236)

Fix PCLAS-982

JCavallo committed:
gtk: Make text fields visually readonly (#241)

Fix PCLAS-1003

JCavallo commented on modules/party/party.py:
Ce hook doit exister en dehors de la boucle.

havaguernicoopengo committed:
sao: Fix border visibility in text forms [CUSTOM] (#240)

Fix PCLAS-993

---------

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

BeatriceSchuster committed:
tryton: Change the translation for the message shown when a required field is empty [CUSTOM] (#237)

Fix #PJAZZ-2244

JCavallo committed:
trytond: Fix translation res_id domain [PREVIEW] (#242)

Fix PCLAS-1009

nicoe committed:
Skip id field when setting default values of failed read calls [PREVIEW] (#247)

* Undo "Consider that records in exception are loaded [PREVIEW]"

* Skip id field when setting default values of failed read calls [PREVIEW]

Fix PCLAS-1006

nicoe committed:
Add missing Widget methods on source editor [CUSTOM] (#246)

Fix PCLAS-1005

nicoe committed:
Do not resize horizontally textarea [CUSTOM] (#245)

Fix PCLAS-996

Commit:
Packaging coog-24.26.2426-rc5

xungongcoopengo committed:
dont revue

Commit:
Packaging coog-24.27.2427-rc2

nicoe committed:
Consider as local ordering expression using simple SQL statements [PREVIEW] (#248)

Rely on the tables dict to choose when to apply the OR-to-UNION optimization [PREVIEW]

Fix PCLAS-1015

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
trytond: Do not crash on searches on None domains (#250)

Fix PCLAS-1047

JCavallo commented on modules/party/party.py:
L'idée ici c'est de récupérer la liste des données qui aura été construite dans la boucle au-dessus, histoire de ne pas avoir à refaire le travail

JCavallo committed:
trytond: Do not over-optimize function active field searches (#251)

Fix PCLAS-1053

Commit:
Packaging coog-24.28.2428-rc2

nicoe committed:
Correctly handle different return values from buttons [CUSTOM] (#243)

Correctly handle different return values from buttons [CUSTOM]

Fix PCLAS-990

nicoe committed:
Reset caches directly after sending the notification [CUSTOM] (#235)

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

JCavallo committed:
sao: Better usage of vertical space [CUSTOM] (#252)

Fix PCLAS-1055

nicoe committed:
Handle the case where the action is None [CUSTOM] (#255)

PCLAS-1007

Commit:
Packaging coog-24.29.2429

havaguernicoopengo committed:
sao: open form in modal when already in modal (#253)

Fix PCLAS-1042

nicoe committed:
Resize columns in sao [PREVIEW] (#164)

Fix PCLAS-786

---------

Co-authored-by: Jean Cavallo <jean.cavallo@hotmail.fr>

Commit:
Packaging coog-24.30.2430-rc3

Commit:
Packaging coog-24.31.2431-rc2

JCavallo committed:
trytond: Add hook to control log creation [CUSTOM] (#259)

Fix PCLAS-1082

JCavallo committed:
sao: Better column resizing (#257)

Commit:
Packaging coog-24.32.2432-rc2

Commit:
Packaging coog-24.33.2433

xungongcoopengo committed:
Merge remote-tracking branch 'origin/master' into prock-1699

xungongcoopengo committed:
revue jean

xungongcoopengo committed:
remove before

xungongcoopengo commented:
@JCavallo Je pense que je vais créer une PR pour tryton
